src/components/angular2-modal/overlay/overlay.service.ts

```diff
@@ -84,7 +84,7 @@ export class Overlay {
     }
 
     let dialog = new DialogRef<any>(this, config.context || {});
-    dialog.inElement = !!config.context.inElement;
+    dialog.inElement = config.context ? config.context.inElement : false;
```

Comment:
`dialog.inElement = config.context ? !!config.context.inElement : false;` throws this error?